commands/src/main/java/org/jclouds/karaf/commands/table/internal/ScriptEngineShellTable.java

```diff
@@ -51,9 +47,7 @@ public String evaluate(Object obj, String expression) {
     try {
       scriptEngine.put(getType(), obj);
       result = String.valueOf(scriptEngine.eval(expression));
-    } catch (Exception ex) {
-      //Ignore
-    }
+    } catch (Exception ignored) {}
     return result;
```

Comment:
Would it make sense to change the default value of the `result` variable to `N/A` or similar? That would make more sense for users than a bunch of empty lines.